spec/factories/whitehall_export/file_attachment_factory.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :whitehall_export_file_attachment, class: Hash do
+    skip_create
+
+    sequence(:id)
+    created_at { Time.zone.now.rfc3339 }
+    updated_at { Time.zone.now.rfc3339 }
+    title { "Some random text file" }
+    accessible { false }
+    isbn { "" }
+    unique_reference { "" }
+    command_paper_number { "" }
+    order_url { "" }
+    price_in_pence { nil }
+    attachment_data_id { 10 }
+    ordering { 0 }
+    hoc_paper_number { "" }
+    parliamentary_session { "" }
+    unnumbered_command_paper { false }
+    unnumbered_hoc_paper { false }
+    attachable_id { 12 }
+    attachable_type { "Edition" }
+    slug { nil }
+    locale { nil }
+    external_url { nil }
+    content_id { SecureRandom.uuid }
+    deleted { false }
+    print_meta_data_contact_address { nil }
+    web_isbn { nil }
+    url { "https://asset-manager.gov.uk/blah/847150/#{filename}" }
+    type { "FileAttachment" }
+    attachment_data do
+      {
+        "id" => 1,
+        "carrierwave_file" => filename,
+        "content_type" => "text/plain",
+        "file_size" => 1057,
+        "number_of_pages" => nil,
+        "created_at" => created_at,
+        "updated_at" => updated_at,
+        "replaced_by_id" => nil,
+        "uploaded_to_asset_manager_at" => updated_at,
+        "present_at_unpublish" => false,
```

Comment:
This value should be `nil` I think? (based on https://whitehall-admin.integration.publishing.service.gov.uk/government/admin/export/document/165362)

Reply:
Hmm, the export I tried (screenshot in the trello card https://trello.com/c/4FF2oprK/1182-import-attachments-into-content-publisher) had a value of `false`, maybe the value can either be false, true or nil.